### PR TITLE
doc: delete room API

### PR DESCRIPTION
**Pull request by ofoucherot**
Add documentation for delete room endpoint: `DELETE /rooms/:roomId`.

### CODE REVIEW

File: docs/references/v2.openapi.json [hunk 1]
```diff
@@ -395,6 +395,29 @@
...
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "422": {
+            "$ref": "#/components/responses/422"
+          }
+        },
+        "operationId": "delete-rooms-roomId",
+        "description": "Deletes a room. A deleted room cannot be accessible from the API or the dashboard and it cannot be restored. \n\n"
```
Comment (nvie) [inline docs/references/v2.openapi.json:420]:
```suggestion
        "description": "Deletes a room. A deleted room is no longer accessible from the API or the dashboard and it cannot be restored. \n\n"
```

**Review by nvie — APPROVED** [review]
Lovely! 👍

**Review by adigau — APPROVED** [review]

**Comment by adigau** [comment]
Makes a lot of sense, I wanted to check if the wording was coherent with the rest and it seems like it 💪

File: docs/references/v2.openapi.json (hunk 1 above)
Comment (ofoucherot) [reply docs/references/v2.openapi.json:420]: Fixed! thank you

**Pull request merged**